tests/test_abi.py

```diff
@@ -119,6 +134,16 @@ def test_selector(self):
         )
         assert abi.selector == "MyMethod(address,string)"
 
+    def test_from_signature(self):
```

Comment:
Same, can this parametrized with a few more examples?

Reply:
at least the following cases for method abi would be nice:

no inputs
1 input
return 1 value
return 2 values